src/plugins/text-input/text-input.yml

```diff
@@ -0,0 +1,19 @@
+textInput:
+  name:
+    en: Text input
+    de: Texteingabe
+  mode:
+    en: Mode
+    de: Modus
+  mode_singleLine:
+    en: Single line
+    de: Einzeilig
+  mode_multiLine:
+    en: Multiple lines
+    de: Mehrzeilig
+  maxLength:
+    en: Maximum length
+    de: Maximale Länge
+  maxLengthInfo:
+    en: Maximum number of characters the user can input (zero means unrestricted)
```

Comment:
How about "Maximum number of characters that can be typed into the text input (zero means unrestricted)" to not mention "the user" as the person setting up the plugin as an editor is also "the user" and I find it can be confusing, maybe?

Reply:
Done, good idea. I also renamed from 'text-input' to 'text-field' as agreed.